### PR TITLE
enable hc5761a USB and SD card

### PR DESCRIPTION
**Pull request by palmxin**
1、HC5761A USB is no power,must enable port"gpio1 7 GPIO_ACTIVE_HIGH"
2、HC5761A has SD card.
3、fix  HC5761A  LED,add"p3led_an"

### CODE REVIEW

**Comment by adrianschmutzler** [comment]
Hi, please read https://openwrt.org/submitting-patches for the formal requirements of a submission at OpenWrt. In the end, I would expect three separate commits:
1. set USB power GPIO
2. enable SD card
3. fix pinctrl

Each of those would have a proper title, commit description of the what and why, and Signed-off-by.

Please make sure you push to this PR with (git push -f ...).

File: target/linux/ramips/dts/mt7628an_hiwifi_hc5761a.dts [hunk 1]
```diff
@@ -32,12 +32,23 @@
 			linux,default-trigger = "phy1tpt";
 		};
 	};
+
+	gpio_export {
+		compatible = "gpio-export";
+		#size-cells = <0>;
+
+		usbpower {
+			gpio-export,name = "usb";
```
Comment (Leo-PL) [inline target/linux/ramips/dts/mt7628an_hiwifi_hc5761a.dts]: Shouldn't tthis be "hiwifi:power:usb"?

File: target/linux/ramips/dts/mt7628an_hiwifi_hc5761a.dts [hunk 2]
```diff
@@ -54,3 +65,7 @@
 		ieee80211-freq-limit = <5000000 6000000>;
 	};
 };
+
+&sdhci {
```
Comment (dengqf6) [inline target/linux/ramips/dts/mt7628an_hiwifi_hc5761a.dts]: As in PR #1500, unless the bug is fixed, blindly enabling SD will render ethernet ports unusable

**Review by dengqf6 — CHANGES_REQUESTED** [review]

File: target/linux/ramips/dts/mt7628an_hiwifi_hc5761a.dts (hunk 1 above)
Comment (xpalm) [reply target/linux/ramips/dts/mt7628an_hiwifi_hc5761a.dts]: I have test on hc5761a,It works Okay。please merge

**Comment by xpalm** [comment]
1、set USB power GPIO
2、enable SD card
3、fix pinctrl
4、fix  PR #1500 bug，enabling SD also ethernet ports usable

**Comment by dengqf6** [comment]
@981213 ping

**Comment by adrianschmutzler** [comment]
@xpalm Please address the formal issues as stated in my first comment.

**Comment by 981213** [comment]
The pinctrl patch isn't a perfect implementation because it doesn't deal with pin reservation properly. So I decided not to apply it.
ref: https://patchwork.ozlabs.org/patch/1131754/

**Comment by xpalm** [comment]
cancle

**Comment by xpalm** [comment]
cancle

**Comment by xpalm** [comment]
1、USB port is nopower, must set"gpio1 7 GPIO_ACTIVE_HIGH"
2、add "p2led_an" to pinctrl

**Comment by 981213** [comment]
Hi!
This pull request has several formal issues as already stated by @adrianschmutzler .
Please carefully read [Submitting patches](https://openwrt.org/submitting-patches) and [Working with GitHub](https://openwrt.org/docs/guide-developer/working-with-github-pr) to fix your pull request.
(Partially translated Chinese version also available here: [提交补丁](https://openwrt.org/start?id=zh/submitting-patches) )
Currently this PR is missing proper commit title/description and a sign-off. And all your current commits should be squashed into one. Note that Github PR description or replies are **not** commit description.

**Comment by 981213** [comment]
closed because there are no response to address the mentioned formal issues for about a month.

**Pull request closed without merging**